swift/Podfile.lock

```diff
@@ -13,4 +13,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: bcab653390ccf2f8e377a439713b531df60e7b01
 
-COCOAPODS: 1.6.1
+COCOAPODS: 1.6.0
```

Comment:
please update your cocoapods version to 1.6.1

Reply:
Hi, here: https://github.com/hsiaosiyuan0/wallet-core/commit/219655ceb4cf5ef4ea32c57634ff3b283708f0cd

Reply:
You should merge hsiaosiyuan0/wallet-core first, it's still old code